src/test/resources/importTest/NoLabel.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<document>
+    <id>ImportTest</id>
+    <version>1.0.0</version>
+    <initials>imp</initials>
+    <title>Import Test</title>
+    <defaultRole>true</defaultRole>
+    <anonymousRole>false</anonymousRole>
+    <role>
```

Comment:
Is it necessary to use roles in this test? The Petri nets for test should contains only minimum logic.

Reply:
Done